docs/fidesops/docs/tutorial/annotate_datasets.md

```diff
@@ -48,15 +48,6 @@ the user by `email`, and from there, travel through other tables linked to `user
     identity: email
 ```
 
-Optional `fidesops_meta` fields:
```

Comment:
Since this is a tutorial (walks through implementing a specific dataset), my opinion we should keep docs re: accepted fields in 1 unified place in docs